test/test_core.py

```diff
@@ -1951,12 +1951,10 @@ def test_em_js(self, args, force_c):
     self.do_core_test('test_em_js.cpp', force_c=force_c)
     self.assertContained("no args returning int", read_file('test_em_js.js'))
 
-  @no_wasm2js('WASM_BIGINT is not compatible with wasm2js')
+  @no_wasm2js('using 64-bit arguments in EM_JS function without WASM_BIGINT is not yet fully supported')
   def test_em_js_i64(self):
-    err = self.expect_fail([EMCC, '-Werror', test_file('core/test_em_js_i64.c')])
+    err = self.expect_fail([EMCC, '-Werror', '-sWASM_BIGINT=0', test_file('core/test_em_js_i64.c')])
     self.assertContained('emcc: error: using 64-bit arguments in EM_JS function without WASM_BIGINT is not yet fully supported: `foo`', err)
-
```

Comment:
Maybe remove the `no_wasm2js` decorator above, and then add a `skipTest("test depends on WASM_BIGINT")` here on this line.

Reply:
Or maybe just revert the comment to `WASM_BIGINT is not compatible with wasm2js` since this test does actually explictly depend on `WASM_BIGINT` right?

Reply:
Yes, it does. I switched it back.